webpack.config.js

```diff
@@ -66,10 +66,11 @@ module.exports = function(opts) {
     use: {
       loader: 'babel-loader',
       options: {
-        presets: ['env'],
+        rootMode: 'upward',
+        presets: ['@babel/preset-env', "@babel/preset-react"],
         plugins: [
-          'transform-class-properties',
-          'transform-object-rest-spread'
+          '@babel/plugin-proposal-class-properties',
+          '@babel/plugin-proposal-object-rest-spread'
         ]
       }
```

Comment:
This config should match the babel.config.js file (eg. it is mising the import plugin - not your fault as it looks like it was missed in a previous version - and does the other config need the rootMode adding?)